spec/features/application_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+require 'capybara/rails'
+
+
+describe ApplicationController do
+
+  feature 'user signs in', :type => :feature, :js => true do
+    let!(:user) { FactoryGirl.create(:user) }
+    scenario 'with valid credentials' do
+      visit new_user_session_path
+      p user.email
+      p user.password
+      fill_in 'Email', :with => user.email
+      fill_in 'Password', :with => user.password
```

Comment:
`fill_in 'Password', :with => user.password`の後に `binding.pry`を挟み以下のようなコマンドを実行しました！

```
> expect(page).to have_content("Password")
=> true

> editor = page.find_by_id('user_password')
=> #<Capybara::Node::Element tag="input" path="//HTML[1]/BODY[1]/DIV[1]/FORM[1]/DIV[2]/INPUT[1]">

> p editor.value
"00000000"
=> "00000000"

> click_button 'Log in'
Capybara::Poltergeist::TimeoutError: Timed out waiting for response to {"id":"7cb7ef8f-6401-4645-bb7c-e31c19b0f694","name":"click","args":[1,158]}. It's possible that this happened because something took a very long time (for example a page load was slow). If so, setting the Poltergeist :timeout option to a higher value will help (see the docs for details). If increasing the timeout does not help, this is probably a bug in Poltergeist - please report it to the issue tracker.
from /Users/yasuomi.takano/.rbenv/versions/2.2.3/lib/ruby/gems/2.2.0/gems/poltergeist-1.9.0/lib/capybara/poltergeist/web_socket_server.rb:99:in `rescue in send'
```

TimeoutErrorエラーみたいですね・・・。